actors/runtime/runtime.go

```diff
@@ -57,9 +57,15 @@ type Runtime interface {
 	// Look up the code ID at an actor address.
 	GetActorCodeCID(addr addr.Address) (ret cid.Cid, ok bool)
 
-	// Randomness returns a (pseudo)random byte array drawing from a
+	// GetRandomnessFromBeacon returns a (pseudo)random byte array drawing from a
 	// random beacon at a given epoch and incorporating reequisite entropy
-	GetRandomness(personalization crypto.DomainSeparationTag, randEpoch abi.ChainEpoch, entropy []byte) abi.Randomness
+	GetRandomnessFromBeacon(personalization crypto.DomainSeparationTag, randEpoch abi.ChainEpoch, entropy []byte) abi.Randomness
```

Comment:
Where do I find the implementation of this in lotus?

Reply:
its not there yet, this needs to happen first